### DISCUSSION TITLE
[3.x]: Propogation settings are confusing

### DISCUSSION THREAD

**Discussion by karensg** [craftcms/cms#10952] [Ideas]
### Description

The propagation settings of fields are confusing.
I have chosen to translate the field for each language. I have 10+ sites on the website. Two of them are: Europe with hreflang `en` en United Kingdom with hrelfang `en-uk`. While they are the same language and I want them to have the same content most of the time, Craft considers them as different ones.

### Steps to reproduce
![image](https://user-images.githubusercontent.com/359682/163951426-149685eb-eabb-43b6-8146-7b52bed71b68.png)

### Expected behavior

To not break anything, I would suggest renaming this translation method to `translate for each language and region` and add a new one `translate for each language`. The latter method would check the first two letters of the defined hreflang.

### Craft CMS version

3.7.38

### PHP version

8.1.4

### Installed plugins and versions

Translations
SeoMatic

**Comment by brandonkelly** [comment]
Converted to a discussion as this is not a bug. The setting is referring to the sites’ “Language” settings, where you would have selected “English (United Kingdom)”.

**Comment by carlcs** [comment]
If you’re looking for a way to configure the field to always have the same content for your “en-GB” and ”en” sites, select “Custom” for the translation method and use this code for the translation key.

```twig
{{ object.site.language|split('-')[0] }}
```
